### ISSUE TITLE
Add a possibility to tag the Wiki Updater commit 

### ISSUE THREAD

**Issue by richieto** [rfennell/AzurePipelines#593]
### Azure DevOps Extension you are using

Wiki Updater

### Where are you running it?

- Azure DevOps Service (VSTS)

### Version of Extension/Task

- Latest public

### Expected behaviour and actual behaviour

The Wiki Updater task is at the end of my release pipeline.
I tag my repositories (git version tag) 
There is currently inconsistency in my tags and changelog update. Given the fact that the last commit comes from Wiki Updater tasks.

My request is, to add the possibility for a lightweight tag in this task. Something like:

GitWikiFuntions.ts L136+ 
```
if (tagRepo) { 
 await git.addTag(tagname); 
 await git.pushTags();
}
```

Let me know your thoughs

Note: I tried some other extensions but they tag artifacts, which finds the last commit for tagging (which is prior the Wiki Updater tasks). So that did not solve my issue.

**Comment by rfennell** [comment]
Seems a reasonable enhancement, I have put it on the backlog. I am not sure when I will get to it, but always happy to consider PRs

**Comment by rfennell** [comment]
I had a bit of time, so added the feature. It is released in 1.7.5. It is basically the logic you suggested with  the wiring to manifest.
